MuonAnalyser/plugins/SliceTestAnalysis.cc

```diff
@@ -1,545 +0,0 @@
-// cd /cms/ldap_home/iawatson/scratch/GEM/CMSSW_10_1_5/src/ && eval `scramv1 runtime -sh` && eval `scramv1 runtime -sh` && scram b -j 10
```

Comment:
i dont think this should be deleted. doesnt @tt8888tt use this?

Reply:
I think its from one of my codes from a long time ago. Im pretty sure DJ can't compile in my home directory

Reply:
Yep Ian is right